openmc/material.py

```diff
@@ -1044,6 +1045,10 @@ def get_activity(self, units: str = 'Bq/cm3', by_nuclide: bool = False):
         by_nuclide : bool
             Specifies if the activity should be returned for the material as a
             whole or per nuclide. Default is False.
+        volume : float, optional
+            Volume of the material if not assigned directly
```

Comment:
Do you think we could make it clearer that if this ```volume``` is not set then ```material.volume``` is used. However this ```volume``` gives users the opportunity to use a different volume to the ```material.volume``` 

Perhaps not quite this wording but what do you think, something along these lines

```suggestion
            Volume of the material to use. If the volume arg is not set then material.volume is used.
```